src/odemis/driver/actuator.py

```diff
@@ -666,6 +667,27 @@ def _get_rot_matrix(self, invert=False):
             rotation *= -1
         return RigidTransform(rotation=rotation).matrix
 
+    def _getAxesRange(self):
+        """
+        Calculate the axes range of position VA from the given range of dep's.
+        """
+        # Calculate rectangular border points on the given range
+        min_x, max_x = self.axes["x"].range
+        min_y, max_y = self.axes["y"].range
+        coordinates = [[min_x, min_y],
+                       [min_x, max_y],
+                       [max_x, min_y],
+                       [max_x, max_y]]
+
+        # Calculate range based on the conversion
+        new_coordinates = list(map(self._convertPosFromdep, coordinates))
+
+        # Update default axes range
+        new_range = numpy.array(new_coordinates, dtype=float)
+        self.axes["x"].range = (min(new_range[:, 0]), max(new_range[:, 0]))
+        self.axes["y"].range = (min(new_range[:, 1]), max(new_range[:, 1]))
```

Comment:
I think this is modifying the original range too because of the definition l 630. You need to make a copy. 
Also, the comment line 627 can now be removed, as you've implemented this TODO.

Reply:
I ll make a copy, this was one of my main worries that it directly affects the dependency and alters its original range.

Reply:
![image](https://github.com/delmic/odemis/assets/37955981/f7e0ee7d-3bcd-4b02-8a12-38ec6cfcce8c)
The first line of the comment has been done, I am not sure what the second and third line means and if needs to be still executed.